docs/src/pages/customization/palette/Palette.js

```diff
@@ -1,6 +1,5 @@
 import * as React from 'react';
-import { createTheme } from '@material-ui/core/styles';
```

Comment:
Needed to do this change to fix https://www.argos-ci.com/mui-org/material-ui/builds/4177

We have two options: 
1. Do nothing about it. The system would depend only on `@material-ui/styled-engine` and use its ThemeContext
2. Add `@material-ui/private-theming` as a dependency to `@material-ui/system`.
For this to work, we should do the following changes:

```diff --git a/packages/material-ui-system/package.json b/packages/material-ui-system/package.json
index cdcd17f47c..f0cc8bc1d0 100644
--- a/packages/material-ui-system/package.json
+++ b/packages/material-ui-system/package.json
@@ -56,6 +56,7 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.4.4",
+    "@material-ui/private-theming": "5.0.0-alpha.35",
     "@material-ui/styled-engine": "5.0.0-alpha.34",
     "@material-ui/types": "6.0.1",
     "@material-ui/utils": "5.0.0-alpha.35",
diff --git a/packages/material-ui-system/src/useTheme.js b/packages/material-ui-system/src/useTheme.js
index 590858bbf9..e73cbff245 100644
--- a/packages/material-ui-system/src/useTheme.js
+++ b/packages/material-ui-system/src/useTheme.js
@@ -1,4 +1,5 @@
 import * as React from 'react';
+import { useTheme as muiUseTheme } from '@material-ui/private-theming';
 import { ThemeContext } from '@material-ui/styled-engine';
 import createTheme from './createTheme';

@@ -9,7 +10,9 @@ function isObjectEmpty(obj) {
 export const systemDefaultTheme = createTheme();

 function useTheme(defaultTheme = systemDefaultTheme) {
-  const contextTheme = React.useContext(ThemeContext);
+  const muiContextTheme = muiUseTheme();
+  const styledEnginecontextTheme = React.useContext(ThemeContext);
+  const contextTheme = muiContextTheme || styledEnginecontextTheme;
   return !contextTheme || isObjectEmpty(contextTheme) ? defaultTheme : contextTheme;
 }

```

Probably it's better to go with Option 2 to avoid surprises in the future.